models/issue_comment.go

```diff
@@ -943,7 +966,12 @@ func fetchCodeCommentsByReview(e Engine, issue *Issue, currentUser *User, review
 	if err := e.In("id", ids).Find(&reviews); err != nil {
 		return nil, err
 	}
+
 	for _, comment := range comments {
+		if err := comment.LoadResolveDoer(); err != nil {
```

Comment:
yes, It willn't be error if user is deleted now, it will load an default user to replace it, but the name to show will be not right , that's then thing what I  think now. maybe shold save user name instead of user id, then it willn't be a problem , do you think so ? Thanks

Reply:
@lunny  If understand what you mean, that's taken care of in `LoadResolveDoer`:

https://github.com/go-gitea/gitea/blob/a2fa3ee0601599e8980597eb2d500c06b96f5338/models/issue_comment.go#L430-L437